### PR TITLE
Added message mapping for CancerType, PrimarySite, Country, and Region.

### PR DESCRIPTION
**Pull request by Buwujiu**

### CODE REVIEW

File: ProgramService.proto [hunk 1]
```diff
@@ -48,7 +40,15 @@ message Program {
   int32 submitted_donors = 7;
   int32 genomic_donors = 8;
   string website = 9;
-  Date date_created = 10;
+  string organizations = 10;
+
+  Date date_created = 11;
+  Date date_updated = 12;
+
+  repeated CancerType cancerTypes = 13;
```
Comment (andricDu) [inline ProgramService.proto]: Lets use snake case for fieldnames. 
`repeated CancerType cancer_types = 13;`

File: ProgramService.proto [hunk 2]
```diff
@@ -92,8 +88,10 @@ message ProgramEgo {
 // Full details of a program
 message ProgramDetails {
   Program program = 1;
-  Organization org = 2;
-  ProgramAdmin admin = 3;
-  ProgramEgo ego = 4;
+  ProgramEgo ego = 2;
+  repeated CancerType cancerTypes = 3;
```
Comment (andricDu) [inline ProgramService.proto]: snake case

File: ProgramService.proto [hunk 3]
```diff
@@ -92,8 +88,10 @@ message ProgramEgo {
...
+  repeated PrimarySite primarySites = 4;
```
Comment (andricDu) [inline ProgramService.proto]: snake case

File: ProgramService.proto [hunk 4]
```diff
@@ -48,7 +40,15 @@ message Program {
   int32 submitted_donors = 7;
   int32 genomic_donors = 8;
   string website = 9;
-  Date date_created = 10;
+  string organizations = 10;
```
Comment (andricDu) [inline ProgramService.proto]: `institutions` was what @alekspejovic specified I think.

File: ProgramService.proto [hunk 5]
```diff
@@ -59,28 +59,24 @@ message ProgramCollection {
...
 
-message OrganizationRequest {
+message PrimarySite {
   string id = 1;
+  string name = 2;
 }
 
-message OrganizationCollection {
-  repeated Organization organizations = 1;
+message Country {
```
Comment (andricDu) [inline ProgramService.proto]: we can probably get rid of this type.

File: ProgramService.proto [hunk 6]
```diff
@@ -48,7 +40,15 @@ message Program {
...
   string website = 9;
-  Date date_created = 10;
+  string organizations = 10;
+
+  Date date_created = 11;
+  Date date_updated = 12;
+
+  repeated CancerType cancerTypes = 13;
+  repeated PrimarySite primary_sites = 14;
+  repeated Country countries = 15;
```
Comment (andricDu) [inline ProgramService.proto]: country can probably just be strings.

File: ProgramService.proto [hunk 7]
```diff
@@ -59,28 +59,24 @@ message ProgramCollection {
...
+message Country {
+  string id = 1;
+  string name = 2;
 }
 
-// Administrator for a program
-message ProgramAdmin {
-  string ego_id = 2;
-  string ego_email = 3;
+message Region {
```
Comment (andricDu) [inline ProgramService.proto]: this can probably also be a string.

File: ProgramService.proto [hunk 8]
```diff
@@ -92,8 +79,8 @@ message ProgramEgo {
 // Full details of a program
 message ProgramDetails {
   Program program = 1;
-  Organization org = 2;
-  ProgramAdmin admin = 3;
-  ProgramEgo ego = 4;
+  ProgramEgo ego = 2;
+  repeated CancerType cancer_types = 3;
+  repeated PrimarySite primary_sites = 4;
```
Comment (rtisma) [inline ProgramService.proto]: As explained above, if this was version 1.0.0, the correct way to do this is
```
message ProgramDetails {
  Program program = 1;	  Program program = 1;
  ProgramEgo ego = 4;
  repeated CancerType cancer_types = 5;
  repeated PrimarySite primary_sites = 6;
}
```

Using this update method, the `ProgramDetails` can be serialized using this NEW protobuf definition above, and still used with code that uses the OLD `ProgramDetails` protobuf definition, hence backwards compatible.

File: ProgramService.proto [hunk 9]
```diff
@@ -48,7 +41,15 @@ message Program {
   int32 submitted_donors = 7;
   int32 genomic_donors = 8;
   string website = 9;
-  Date date_created = 10;
+  string institutions = 10;
```
[the remark this answers is not in the document]
Comment (Buwujiu) [reply ProgramService.proto]: Thanks for the explanation!

**Pull request merged**